checkpoint/handler.go

```
@@ -95,7 +95,11 @@ func handleMsgCheckpoint(ctx sdk.Context, msg types.MsgCheckpoint, k Keeper, con
 	// Calculate new account root hash
 	dividendAccounts := k.sk.GetAllDividendAccounts(ctx)
 	k.Logger(ctx).Debug("DividendAccounts of all validators", "dividendAccounts", dividendAccounts)
-	accountRoot, err := types.GetAccountRootHash(dividendAccounts)
+	var accountRoot []byte
+	accountRoot, err = types.GetAccountRootHash(dividendAccounts)
+	if err != nil {
+		k.Logger(ctx).Error("handleMsgCheckpoint | GetAccountRootHash", "Error", err)
```

We should return an error if account hash an error